Corrected link in talking point about order of operations. (#580)
exercism/website-copy: tracks/csharp/exercises/leap/mentoring.md

### Talking points

> - Students often use parentheses, but this is not always necessary, as the [precedence](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/#conditional-and-operator) of the `&&` and `||` operators is based on the order in which they appear in the code.

Author: ```suggestion
- Students often use parentheses, but as the `&&` operator has a [higher precedence](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/#conditional-and-operator) than the `||` operator, this is not necessary.
```
